app/src/main/java/org/mozilla/tv/firefox/framework/FrameworkRepo.kt

```diff
@@ -16,10 +16,19 @@ import android.view.accessibility.AccessibilityManager
  */
 class FrameworkRepo @VisibleForTesting(otherwise = PRIVATE) constructor() {
 
+    private var wasInitCalled = false
+
     private val _isVoiceViewEnabled = MutableLiveData<Boolean>()
     val isVoiceViewEnabled: LiveData<Boolean> = _isVoiceViewEnabled
 
+    /**
+     * Initializes this repository.
+     * @throws IllegalStateException when called twice.
+     */
     fun init(accessibilityManager: AccessibilityManager) {
+        if (wasInitCalled) throw IllegalStateException("FrameworkRepo.init unexpectedly called twice")
```

Comment:
I think I would prefer that this be a Sentry call, but I don't think we have a team policy regarding this yet so I won't block on it.

Reply:
I think you're right this could have been a sentry call (though it'd be a little harder to test). However, I don't expect it to happen and I want to minimize the number of changes to this PR after the review so I'm going to land it as is.